frontend/src/lib/components/EditableText.svelte

```diff
@@ -127,7 +127,7 @@
       on:keypress={startEditing}
     >
       {#if value}
-        <span class="mr-2 whitespace-pre-wrap text-primary-content">{value}</span>
```

Comment:
Here's the proof that `text-primary-content` is a "real" class 😄, but it certainly isn't the right one.
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/c857bdec-3974-4fa4-b95c-d8352036c39f)

I've been thinking about this. We haven't put any thought into our "color palette" / "brand" yet. I think the right thing to do here for now, is just decide for each element if it's primary, secondary, accented or just normal/neutral and simply ignore the actual color for now.

I would lean towards:
- Make the project name and project values (code, last commit date) secondary (e.g. `text-secondary`) (I don't think they _quite_ deserve our primary brand color 🤷). I believe this will make them a sickening pick, but that's alright (or see below).
- Make the project description a neutral text color (either no css class or text-neutral I think). Just because it's so long.
- Probably make sure the pencil icons don't inherit the secondary color, but instead use a neutral color, because they're not part of the meaningful content.

If the pink is so aweful that you think it will be bad for our health as developers before we get around to working on our color palette, then I would suggest:
1) Change our theme config in `tailwind.config.cjs` so that we only have 1 (e.g. just toss the dark one as well as any customizations)
2) Pick a less aggressive theme from here: https://daisyui.com/docs/themes/#. (Maybe winter, aqua or corporate?)

Reply:
I could be doing something wrong 🤔, but text-secondary seems to make it blend into the background:
![image](https://github.com/sillsdev/languageforge-lexbox/assets/61605641/f94ecfec-5be8-4265-af7f-2b172f43a739)
(last commit is text-secondary while code is text-primary)
It looks like both text-primary and text-secondary are being set forcibly in ```tailwind.config.cjs``` like you mentioned:
```
daisyui: {
    themes: [
      {
        light: {
          ...require('daisyui/src/colors/themes')['[data-theme=light]'],
          primary: '#104060',
          secondary: '#0a2440',
        },
      },
      {
        dark: {
          ...require('daisyui/src/colors/themes')['[data-theme=dark]'],
          primary: '#104060',
          secondary: '#0a2440',
        },
      },
    ],
  },
  ```
  This could explain why they aren't automatically changing when we switch from light to dark (I am new to DaisyUI so I'm not entirely sure this is how that works), I can try changing from the default theme for now if you are fine with me removing them.

Reply:
@myieye Ok so if I change the from the default themes and remove the ```primary``` and ```secondary``` defaults here is what it looks like.
dark-mode (business):
![image](https://github.com/sillsdev/languageforge-lexbox/assets/61605641/5d2db044-e62d-4b77-b386-a6af22187c33)
and light mode (winter):
![image](https://github.com/sillsdev/languageforge-lexbox/assets/61605641/00a9a381-707e-47ad-9013-8cf21d778215)